### PR TITLE
Correct & specify some doc for Windows

### PR DESCRIPTION
**Pull request by kalbermattenm**
Title says it all, sorry for not having seen when I did submit the PR...

### CODE REVIEW

**Comment by sbrunner** [comment]
Thanks, the Travis error come from #1414

**Comment by sbrunner** [comment]
You can do a rebase to fix the Travis error :-)

**Comment by kalbermattenm** [comment]
Oups... That was not quite what I wanted to do...

**Comment by kalbermattenm** [comment]
I think that I will just start all over again, because it is too messy right now...

**Comment by sbrunner** [comment]
just do:
```
git fetch upstream
git rebase upstream/master
git push ... -f
```
and it should me ok :-)

**Comment by sbrunner** [comment]
`upstread` is the camptocamp remote repository :-)

**Comment by sbrunner** [comment]
Nice :-)

**Comment by kalbermattenm** [comment]
OK... thanks a lot @sbrunner... I was messing the thinks up by using the interactive rebase mode...

File: doc/integrator/upgrade_application.rst [hunk 1]
```diff
@@ -54,7 +54,9 @@ Easy upgrading an application
 
         ``git push <remote> <branch>``
 
-    * Follow the Windows instructions hereunder
+    * Follow the Windows instructions hereunder, note that you will have to use
+    ``.build\venv\Scripts\...`` instead of ``.build\venv\bin\...`` in all given
+    commands
```
Comment (sbrunner) [inline doc/integrator/upgrade_application.rst]: Indent issue :-)

**Comment by kalbermattenm** [comment]
Travis and me is all about love-hate... :cry:

**Comment by sbrunner** [comment]
Thanks @kalbermattenm  :-)

**Pull request merged**